moped-editor/src/views/projects/projectsListView/ProjectsListViewTable.js

```diff
@@ -558,7 +565,7 @@ const ProjectsListViewTable = ({ title, query, searchTerm, referenceData }) => {
                     Body: props => {
                       const indexedData = data["project_list_view"].map(
                         (row, index) => ({
-                          tableData: { id: index },
+                          tableData: { id: index, uuid:row.project_id },
```

Comment:
Curious about this addition here - I couldn't figure out what the uuid is for and I couldn't find anything about it in the material-table docs. It's also been a while since I've worked with these components so bear with me 😅

Reply:
well now i'm curious, too! it seems to fix a unique key prop warning. i checked material-table docs (original and fork) and found nothing about this. Chia, the ultimate 🕵️‍♀️ ??